packages/editor/src/components/Editor.js

```diff
@@ -135,27 +102,50 @@ class BlockEditor extends Component {
   }
 
   render() {
+    const { plugins, theme, components } = this.props
+    const allComponents = {
+      ...defaultBlocks,
```

Comment:
Since the JSX blocks plugin duck types based on `propertyControls`, I kept `components` as a single prop – could split that out if it makes more sense to be explicit in the API